src/worldclock.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
Components and their files should always be spelled with Capital first letter :)!